src/daft-core/src/array/ops/list.rs

```diff
@@ -112,6 +112,67 @@ impl ListArray {
             Box::new(arrow2::array::Utf8Array::from_iter(result)),
         )))
     }
+
+    fn get_children_helper(
+        &self,
+        idx_iter: &mut impl Iterator<Item = Option<i64>>,
+        default: &Series,
+    ) -> DaftResult<Series> {
+        let default = default.cast(self.child_data_type())?;
```

Comment:
`default` isn't a reserved keyword in Rust but the syntax highlighting is different on it. Weird.